src/irmin-pack/unix/chunked_suffix.ml

```diff
@@ -50,17 +61,26 @@ module Make (Io : Io.S) (Errs : Io_errors.S with module Io = Io) = struct
       open_chunk:
         (chunk_idx:int -> is_appendable:bool -> (Ao.t, open_error) result) ->
       (t, [> open_error ]) result
+
+    val add_new_appendable :
+      open_chunk:
+        (chunk_idx:int -> is_appendable:bool -> (Ao.t, add_new_error) result) ->
```

Comment:
Can `add_new_error` still be `open_error` here?

Reply:
I tried, it does not compile, I'm not sure why.